assets/scss/application.scss

```diff
@@ -11,8 +11,36 @@ $govuk-page-width: $moj-page-width;
 @import './components/summary-card';
 @import './components/card';
 @import 'assets/scss/local';
+@import './components/sortable-table';
 @import "../../node_modules/govuk-frontend/dist/govuk/base";
 @import "../../node_modules/@ministryofjustice/frontend/moj/components/sortable-table/_sortable-table";
-@import './components/sortable-table';
 
+// TM styles
+.secondary-text {color: $govuk-secondary-text-colour; /* Using a GOV.UK Design System color */}
+.strong { font-weight: 900;}
```

Comment:
Did you try using the `govuk-!-font-weight-bold` class, or is it not bold enough?

Reply:
I think Tim wanted something slightly lighter than the usual bold